packages/ActionBar/src/hooks/useSort/sort.js

```diff
@@ -0,0 +1,37 @@
+import moment from "moment";
+
+function compareString(a, b) {
+  return a.localeCompare(b);
```

Comment:
Regular `.localeCompare` might be slow. 

For sorting large arrays it is recommended to use `Intl.Collator` (but if the array size is small, then `localeCompare` should be OK).

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Collator

```js
const compareFunctionFallback = (a, b) => (a < b ? -1 : a > b ? 1 : 0);

function getCompareFunc() {
  if (!Intl || !Intl.Collator) {
    return compareFunctionFallback;
  }

  return new Intl.Collator(I18n.locale, {
    sensitivity: 'base',
  }).compare;
}

let alphaSortCompareFunction;

export function alphaSortAsc(a, b) {
  if (!alphaSortCompareFunction) {
    alphaSortCompareFunction = getCompareFunc();
  }

  return alphaSortCompareFunction(a, b);
}

export function alphaSortDesc(a, b) {
  if (!alphaSortCompareFunction) {
    alphaSortCompareFunction = getCompareFunc();
  }

  return alphaSortCompareFunction(b, a);
}
```

@nahumzs

Reply:
We should definitely try to implement this, we suffered with localeCompare when having thousands of rows.